templates/songs.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang=en>
+<head>
+<meta http-equiv=content-type content="text/html; charset=utf-8" />
+<meta name=viewport content="width=device-width, initial-scale=1">
+<link rel=icon type=image/x-icon href=/static/hacky-day.png />
+<link rel=stylesheet type=text/css href=/static/index.css />
+
+<title>hack·star - Song Management</title>
+<script>
+function deleteSong(songId) {
+  if (confirm('Are you sure you want to delete this song? This action cannot be undone.')) {
+    fetch('/songs/' + songId + '/delete', {
+      method: 'DELETE'
+    }).then(response => {
+      if (!response.ok) {
+        alert('Error deleting song');
+        return;
+      }
+      // Remove the song element from the DOM
+      document.getElementById('song-' + songId).remove();
+      
+      // Update the songs count and show "no songs" message if all songs are deleted
+      const remainingSongs = document.querySelectorAll('.song-item').length;
+      if (remainingSongs === 0) {
+        const songsContent = document.getElementById('songs-content');
+        if (songsContent) {
+          songsContent.innerHTML = '<div class="no-songs"><p>No songs found in the database.</p><p><a href="/static/upload.html">Upload some songs →</a></p></div>';
+        }
+      }
+    }).catch(error => {
+      alert('Error deleting song: ' + error);
+    });
+  }
+}
+</script>
+</head>
+<body>
+
+	<header class=top>
+		<a><img alt="hacky·day"></a>
+        Song Management
+	</header>
+
+	<main>
+    <div id="songs-container">
+      <h2>Songs</h2>
+      
+      {% if songs %}
+      <div id="songs-content">
+        {% for song in songs %}
+        <div class="song-item" id="song-{{ song.hex_id }}">
+          <div class="song-metadata">
+            {% if song.cover %}
+            <div class="song-cover">
+              <img src="{{ song.cover }}" alt="Cover" />
+              {% if song.import_status == 'finished' %}
+              <a href="/song/{{ song.hex_id }}" class="play-overlay" target="_blank" title="Play song">
+                <div class="play-icon">▶</div>
+              </a>
+              {% endif %}
+            </div>
+            {% endif %}
+            <div class="song-info">
+              <div class="song-title">{{ song.title }}</div>
+              <div class="song-artist">{{ song.artist }}</div>
+              <div class="song-year">{{ song.release_date }}</div>
+              <div class="song-id">ID: {{ song.hex_id }}</div>
+            </div>
+          </div>
+          
+          <div class="song-status">
+            <div class="import-status import-status-{{ song.import_status }}">
+              Status: {{ song.import_status }}
+            </div>
```

Comment:
Instead of showing the import status (downloading, processing) here, can we put an icon in place of the cover? This saves space and there is no cover while processing anyway.